lib/boot.js

```diff
@@ -64,12 +64,15 @@ const client = simperium(
         objectStore.createIndex('alphabetical', 'contentKey');
       },
     },
+    preferences: function(objectStore) {
+      console.log('Configure preferences', objectStore); // eslint-disable-line no-console
+    },
     tag: function(objectStore) {
       console.log('Configure tag', objectStore); // eslint-disable-line no-console
     },
   },
   'simplenote',
-  40
+  41
```

Comment:
Can we make this a constant like `dbVersion` or something so that future devs can grok this better?

Reply:
Good point. There are a lot of arguments here, so I changed it into an object to make it clearer all around. 95f94cd